docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -124,6 +124,97 @@ elastic-agent-fj2z9             1/1     Running   0          81m
 elastic-agent-hs4pb             1/1     Running   0          81m
 ------------------------------------------------
 
+[discrete]
+== Red Hat OpenShift configuration
+
+If you are using Red Hat OpenShift, you need to specify additional settings in
+the manifest file and enable the container to run as privileged.
+
+. In the manifest file, modify the `agent-node-datastreams` ConfigMap and adjust inputs:
+`kubernetes-cluster-metrics` input:
+- If used https to access kube-state-metrics, to all `kubernetes.state_*` dataset should be added:
++
+[source,yaml]
+------------------------------------------------
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  ssl.certificate_authorities:
+    - /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+------------------------------------------------
+`kubernetes-node-metrics` input:
```

Comment:
I won't have time to review this until next week, but I wanted to mention that the formatting in this section doesn't look quite right. There's some useful documentation about complex list items here that might help you: https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/

I think their examples are a little confusing, but I personally find it easier to use  open blocks to create a complex list. For example:

```
. I'm a numbered list item.
+
--
I'm a bunch of content nested under the list item.

I can be multiple paragraphs, code examples, lists, and so on.

I cannot contain another open block.
--
```

Reply:
Thank you for the hint! I've updated formatting